### ISSUE TITLE
dist dir loads petstore swagger in v3.0.9 (bug)

### ISSUE THREAD

**Issue by jb08** [swagger-api/swagger-ui#3059]
When I use v.3.0.6 and change the target url to my swagger.json, I receive my API in swagger-ui:
```  
const ui = SwaggerUIBundle({
    url: "http://localhost:8090/api-doc/swagger.json",
    dom_id: '#swagger-ui',
    presets: [
      SwaggerUIBundle.presets.apis,
      SwaggerUIStandalonePreset
    ],
    plugins: [
      SwaggerUIBundle.plugins.DownloadUrl
    ],
    layout: "StandaloneLayout"
  })
```

When I tried upgrading to v3.0.9 and changed the url (same as above), I receive the petstore swagger-ui, rather than my API:

<img width="474" alt="screen shot 2017-05-09 at 10 54 39 am" src="https://cloud.githubusercontent.com/assets/8146460/25860058/f6d1bd2a-34a5-11e7-9287-a2e56fb6ec06.png">

(this is a scalatra app using jetty as server)

**Comment by skors** [comment]
having the same issue. Loading with the config as GET Parameter still works, as a workaround. e.g 'http://localhost/swagger/index.html?url=http://localhost/swagger/swagger.json'

**Comment by skors** [comment]
Downgrading to v3.0.8 works with the correct, expected behaviour.

**Comment by webron** [comment]
Duplicate of #3045.

**Comment by gmartinezsan** [comment]
@skors @jb08 I downloaded the latest source code  from swagger-editor and swagger-ui. All worked fine but cannot get the swagger ui to load a local file instead the default Petstore example.
Is this still an issue?
Thanks in advance,
-gabriela

**Comment by jb08** [comment]
@gmartinezsan modify this [line](https://github.com/swagger-api/swagger-ui/blob/master/dist/index.html#L77) to point to your swagger json.

**Comment by gmartinezsan** [comment]
@jb08 I copied my swagger.json to the dist folder and changed that line too.

my /swagger/swagger-editor/swagger-ui/dist/index.html file has:

window.onload = function() {

  // Build a system
  const ui = SwaggerUIBundle({
    url: "swagger.json",
    dom_id: '#swagger-ui',
    deepLinking: true,
    presets: [
      SwaggerUIBundle.presets.apis,
      SwaggerUIStandalonePreset
    ],
    plugins: [
      SwaggerUIBundle.plugins.DownloadUrl
    ],
    layout: "StandaloneLayout"
  })

  window.ui = ui
}}

But still getting the petstore example.

Any other thing I should change?

Thanks,

**Comment by jb08** [comment]
your service should be serving the `swagger json`, like the one at the [petstore](http://petstore.swagger.io/v2/swagger.json). Then, `url` should be a full url like `localhost:5000/swagger.json`. If you paste in the json to the dist directory, that ?may possibly work, but you would still need a full url, like `localhost:5000/api-docs/swagger.json`

**Comment by jb08** [comment]
^^@gmartinezsan the issue above was for bug in swagger-ui `v3.0.9` that has long since been fixed by the current release `v3.3.2`

**Comment by gmartinezsan** [comment]
Thanks a lot @jb08 !! everything working now with the url.
